content/documentation.rst

```diff
@@ -62,20 +62,71 @@ semicolons, rather than after the Nth column.
     long lines.
 
 
-Writing User's Guides
-=====================
+About the Documentation Sections
+================================
+
+**Introduction**
+
+High level introduction to the software.
+
+
+**Installation and Upgrade Instructions**
+
+System requirements, installation, installation validation/procedures, upgrade
+procedures, uninstallation instructions.
+
+
+**Getting Started with SFTPPlus**
+
+Prerequisite to this page is the installation section.
+
+Contains information about the initial, basic configurations that come with the
+installation package and first account configurations.
+
+
+**Configuration Instructions**
+
+Should contain information to the general configuration principle and
```

Comment:
It feels strange to see an introductory sentece starting with "Should"

I was expecting something like

The configuration section contains information...